specification/cluster/remote_info/ClusterRemoteInfoResponse.ts

```diff
@@ -21,10 +21,11 @@ import { DictionaryResponseBase } from '@_types/Base'
 import { integer, long } from '@_types/Numeric'
 import { Time } from '@_types/Time'
 
-export class Response extends DictionaryResponseBase<
-  string,
-  ClusterRemoteInfo
-> {}
+export class Response {
+  '200': {
+    body: DictionaryResponseBase<string, ClusterRemoteInfo>
```

Comment:
Can we remove `DictionaryResponseBase` from the spec and always use `Dictionary`?

```suggestion
    body: Dictionary<string, ClusterRemoteInfo>
```

Reply:
If we are not extending anymore and we directly specify the body type in the property of the response we don't need the `DictionaryResponseBase` which is only there to be extended from.

In line with the rest of the changes I'd say yes, we can.

Reply:
The .NET client will require such responses to derive from a `DictionaryResponseBase` base class which includes custom JSON formatting and a backing dictionary which derived type expose through a meaningful name. I don't think this change is too much a problem though as my generator could likely infer that for any responses where the body is a Dictionary type. I'm not currently generating these responses for the v8 alpha as they're on the long TODO list!